elixir/apps/domain/lib/domain/network.ex

```diff
@@ -4,7 +4,7 @@ defmodule Domain.Network do
 
   @cidrs %{
     ipv4: %Postgrex.INET{address: {100, 64, 0, 0}, netmask: 10},
-    ipv6: %Postgrex.INET{address: {64_768, 8_209, 4_369, 0, 0, 0, 0, 0}, netmask: 106}
+    ipv6: %Postgrex.INET{address: {0xfd00, 0x2021, 0x1111, 0, 0, 0, 0, 0}, netmask: 106}
```

Comment:
@AndrewDryga I thought this might a tiny bit easier to read. WDYT?

Reply:
Probably easier to read but I'm trying to use the value that we will get back from DB so if we grep on project we can find the roots of it

Reply:
Kk will revert.